src/RMPI/mpidata.jl

```diff
@@ -59,6 +59,9 @@ function Base.show(io::IO, md::MPIData)
     end
 end
 
+function VectorInterface.zerovector(md::MPIData)
+    return MPIData(zerovector(md.data), md.comm, md.root, md.s)
+end
```

Comment:
In how far should `MPIData` conform to `VectorInterface`? Maybe not if we are not planning to use it for `KrylovKit` or similar. Do we then still need the `zerovector` method?